app/views/layouts/application.html.erb

```diff
@@ -7,16 +7,17 @@
     <%= csp_meta_tag %>
     <%= stylesheet_pack_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <script src="https://kit.fontawesome.com/e03d7ee539.js" ></script>
```

Comment:
maybe I don't understand this very well, but is this the best way to add `fontawesome` to the project?

Reply:
We don't focus to add icons manually svg because we want to have advantage to reduce the band width, what do you think exist another better option that you want share with us?